Readme.md

```diff
@@ -36,6 +38,27 @@ Alternately you could start INSTEAD with preloaded game, for example:
 $ sdl-instead path/to/old-mixer-tales
 ```
 
+### JavaScript INSTEAD Client
+
+There's an [instead-js] project we also support. To run it, follow the
```

Comment:
THERE CAN BE ONLY ONE— Erm, sorry; I mean, there is only one "instead-js" project, so definitive article should be used.

"Running" a project means managing it; not what you intended to say, I guess.

I suggest to rewrite these as follows: "The game can also be run inside a webclient provided by the [intead-js] project. To do so, …".